Makefile

```diff
@@ -6,23 +6,25 @@ VERSION = 3.0.0
 SANITY_TEST_ARGS ?= --docker --color
 UNITS_TEST_ARGS ?= --docker --color
 PYTHON_VERSION ?= `python3 -c 'import platform; print("{0}.{1}".format(platform.python_version_tuple()[0], platform.python_version_tuple()[1]))'`
+INSTALL_PATH ?= ~/openshift/ansible_collections
```

Comment:
Why not use a tmp directory that gets cleaned up?

Reply:
we can do that, but my intention was to keep it simple as much as possible

Reply:
done